CMS-Enterprise/easi-app
# [NOREF] ArchiveSystemIntake - convert Rest to GQL

# NOREF

## Description

- convert `archiveSystemIntake` to GQL

## How to test this change

- create a system intake but _do not_ submit (leave as draft)
- nav to the governance task list and click `Remove your request`
- confirm it in the modal
- confirm the intake is gone and that you were redirected to home page

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
